tslint.json

```diff
@@ -5,9 +5,18 @@
     ],
     "jsRules": {},
     "rules": {
-        "interface-name": [true, "never-prefix"],
+        "interface-name": [
+            true,
+            "never-prefix"
+        ],
         "max-classes-per-file": false,
-        "object-literal-sort-keys": false
+        "object-literal-sort-keys": false,
+        "ordered-imports": [
+            true,
+            {
+                "import-sources-order": "any"
```

Comment:
If you don't like this rule, you can separate the imports by an empty line to allow a specific order. In other words, this rule only requires order within "groups" of imports.